ios/Sources/GutenbergKit/Sources/EditorViewControllerDelegate.swift

```diff
@@ -1,6 +1,7 @@
 import Foundation
 
 #if canImport(UIKit)
+@MainActor
```

Comment:
Added in https://github.com/wordpress-mobile/GutenbergKit/pull/186/commits/bd8f3c08d493ee2fcee53bb9efeb7f4cf5e2bf17 to address conformance warning.